security/s2a/internal/crypter/testutil/common.go

```diff
@@ -15,7 +15,7 @@ const (
 type CryptoTestVector struct {
 	ID                                          int
 	Key, Plaintext, Ciphertext, Tag, Nonce, Aad []byte
-	Result, Comment                             string
+	Result, Desc                                string
```

Comment:
Please add the description first, unless this order is mandated by the JSON structure.

Reply:
Done